src/h5web/vis-packs/core/shared/PanZoomMesh.tsx

```diff
@@ -72,7 +72,6 @@ function PanZoomMesh(): ReactElement {
     (evt: WheelEvent) => {
       const factor = evt.deltaY > 0 ? ZOOM_FACTOR : 1 / ZOOM_FACTOR;
 
-      // eslint-disable-next-line no-param-reassign
       camera.zoom = Math.max(1, camera.zoom * factor);
```

Comment:
Why can you disable this now ?

Reply:
My guess is, the rule is smarter. It's meant to forbid `camera = ...`, not mutating the parameter like we are doing here.